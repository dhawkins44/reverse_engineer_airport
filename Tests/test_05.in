10 17 10 5
0 9
0 0
1 1
2 1
3 1
4 1
5 1
6 1
7 1
8 1
9 1
0 1 10
0 2 10
0 3 10
0 4 10
0 5 10
0 6 10
0 7 10
0 8 10
0 9 10
1 2 10
2 3 10
3 4 10
4 5 10
5 6 10
6 7 10
7 8 10
8 9 10