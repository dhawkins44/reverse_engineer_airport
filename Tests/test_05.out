No suitable path from 0 to 9 exists
